pkg/nanopb/Makefile.include

```diff
@@ -0,0 +1 @@
+INCLUDES += -L$(PKGDIRBASE)/nanopb/ -I$(PKGDIRBASE)/nanopb/
```

Comment:
This is the place to include the Makefile.template so that the rules for processing proto files are executed by the top level make.

Reply:
> The processing of the proto files should be moved outside the package. Currently our handling of generated files is less than optimal, I ran into similar issues with the Lua package, and I'm still not satisfied with the results.
> 
> The package's makefile should be written so that it does nothing if it should do nothing.

Okay, I take a look into the Lua package. Thanks for the info. I just don't want to have this in the applications Makefile. The user shouldn't maintain this. Should just run out of the box after including the package into the project.